src/OrleansRuntime/Startup/StartupBuilder.cs

```diff
@@ -14,7 +14,7 @@ namespace Orleans.Runtime.Startup
     /// <summary>
     /// Configure dependency injection at startup
     /// </summary>
-    public class ConfigureStartupBuilder : IStartupBuilder
+    internal class StartupBuilder
     {
         internal static IServiceProvider ConfigureStartup(string startupTypeName)
```

Comment:
I'd like to be able to configure the startup type in a non-stringly fashion, passing it by-instance. This helps when I want to use the same container between multiple libraries/frameworks. Is that too big of a change for this PR?